assignment5.txt

```diff
@@ -0,0 +1,16 @@
+Aly Milich
+
+1. int main(){
+	int a, c;
+	float b;
+
+	fxn(a, b, c);
+
+	return 0;
+}
+
+The variable types all need to match the parameters of fxn which is why a and c are declared as ints while b is a float. From there, you can call the function by using the name and then what you would like to pass through (in this case a, b, c).
+
+2. Iteration is the repetition of a command through the usage of loops and statements. Recursion uses repeated actions to carry out a procedure by calling itself and requires a base case to evaluate from. Both are useful and can be beneficial in different scenarios. Sometimes it is easier for the computer to calculate an output through iteration and sometimes it is more efficient to use past values/calculations to produce the next output (recursion). For example, fibonacci numbers make much more sense to evaluate recursively while the sum of the natural numbers is slightly more intuitive iteratively.
```

Comment:
It's important to note that recursion is less memory efficient. now that we're learning about memory management and data structures, you can further understand this by understanding that each time a function is called, a stack is initialized. So having a recursive function can be very memory ineffective.